examples/spring-boot-draft/README.md

```diff
@@ -0,0 +1,107 @@
+SpringBoot Demo for using Jib
+
+Tips: this is just a draft, please contribute if you have good suggestion.
```

Comment:
Let's put this in `##` so that it is it clear it is in draft state.

s/contribute if you have good suggestion/contribute if you have suggested edits